--- .tmux.conf.local.u1804dvm	2021-11-23 09:36:15.445298087 -0500
+++ origin/.tmux.conf.local	2021-11-23 09:40:17.139590167 -0500
@@ -4,9 +4,6 @@
 #         without any warranty.
 #         Copyright 2012— Gregory Pakosz (@gpakosz).
 
-# -- custom environment initialization -----------------------------------------
-# Create the tmux history folder
-run -b 'mkdir -p ~/.tmux.history 2 > /dev/null'
 
 # -- navigation ----------------------------------------------------------------
 
@@ -197,10 +194,10 @@
 tmux_conf_theme_left_separator_sub="|"
 tmux_conf_theme_right_separator_main=""
 tmux_conf_theme_right_separator_sub="|"
-#tmux_conf_theme_left_separator_main="\uE0B0"  # /!\ you don't need to install Powerline
-#tmux_conf_theme_left_separator_sub="\uE0B1"   #   you only need fonts patched with
-#tmux_conf_theme_right_separator_main="\uE0B2" #   Powerline symbols or the standalone
-#tmux_conf_theme_right_separator_sub="\uE0B3"  #   PowerlineSymbols.otf font, see README.md
+#tmux_conf_theme_left_separator_main='\uE0B0'  # /!\ you don't need to install Powerline
+#tmux_conf_theme_left_separator_sub='\uE0B1'   #   you only need fonts patched with
+#tmux_conf_theme_right_separator_main='\uE0B2' #   Powerline symbols or the standalone
+#tmux_conf_theme_right_separator_sub='\uE0B3'  #   PowerlineSymbols.otf font, see README.md
 
 # status left/right content:
 #   - separate main sections with "|"
@@ -323,25 +320,22 @@
 #   - false (default)
 # on macOS, this requires installing reattach-to-user-namespace, see README.md
 # on Linux, this requires xsel or xclip
-tmux_conf_copy_to_os_clipboard=true
+tmux_conf_copy_to_os_clipboard=false
 
 
 # -- user customizations -------------------------------------------------------
 # this is the place to override or undo settings
 
 # increase history size
-set -g history-limit 1000000
-
-# Command prompt history for session. Saved on exit.
-set -g history-file "~/.tmux.history/session-#S"
+#set -g history-limit 10000
 
 # start with mouse mode enabled
-set -g mouse on
+#set -g mouse on
 
 # force Vi mode
 #   really you should export VISUAL or EDITOR environment variable, see manual
-set -g status-keys vi
-set -g mode-keys vi
+#set -g status-keys vi
+#set -g mode-keys vi
 
 # replace C-b by C-a instead of using both prefixes
 # set -gu prefix2
@@ -353,10 +347,6 @@
 # move status line to top
 #set -g status-position top
 
-# Save tmux history for the current session, window, and pane 
-#bind P command-prompt -p 'save history to filename:' -I '~/tmux.history'                  'capture-pane -S -32768 ; save-buffer %1 ; delete-buffer'
-bind H command-prompt -p 'save history to filename:' -I '~/.tmux.history/#S.w-#W_#I.p-#P' 'capture-pane -S - ; save-buffer %1 ; delete-buffer'
-
 
 # -- tpm -----------------------------------------------------------------------
 
@@ -377,6 +367,17 @@
 #   - false
 tmux_conf_update_plugins_on_reload=true
 
+# by default, reloading the configuration will uninstall tpm and plugins when no
+# plugins are enabled
+#   - true (default)
+#   - false
+tmux_conf_uninstall_plugins_on_reload=true
+
+# /!\ the tpm bindings differ slightly from upstream:
+#   - installing plugins: <prefix> + I
+#   - uninstalling plugins: <prefix> + Alt + u
+#   - updating plugins: <prefix> + u
+
 # /!\ do not add set -g @plugin 'tmux-plugins/tpm'
 # /!\ do not add run '~/.tmux/plugins/tpm/tpm'
 
